contracts/Example3.sol

```diff
@@ -1,14 +1,15 @@
 pragma solidity ^0.8.4;
 
 contract Example3 {
-    uint256 marginPercentage = 30;
+    uint8 marginPercentage = 30;
+    uint8 marginInverse = 100 - marginPercentage;
```

Comment:
Since storage is costly both to store and to access, it is better if this is computed rather than being stored